data/com/mojang/math/Quaternion.mapping

```diff
@@ -1,5 +1,42 @@
 CLASS com/mojang/math/Quaternion
+	METHOD <init> (FFFF)V
+		ARG 1 i
+		ARG 2 j
+		ARG 3 k
+		ARG 4 r
+	METHOD <init> (Lcom/mojang/math/Quaternion;)V
+		ARG 1 other
+	METHOD <init> (Lcom/mojang/math/Vector3f;FZ)V
+		ARG 1 vector
+		ARG 2 angle
+		ARG 3 degrees
+	METHOD cos (F)F
+		ARG 0 value
+	METHOD equals (Ljava/lang/Object;)Z
+		ARG 1 other
+	METHOD fromXYZ (FFF)Lcom/mojang/math/Quaternion;
+		ARG 0 x
+		ARG 1 y
+		ARG 2 z
+	METHOD fromXYZ (Lcom/mojang/math/Vector3f;)Lcom/mojang/math/Quaternion;
+		ARG 0 vector
```

Comment:
`vector` -> `radiansVector`. Or maybe just leave a comment saying what kind of a vector it is (values held).

Reply:
Added a comment instead, since the method names imply that unless specified, values are assumed to be in radians.